moderncvbodyi.sty

```diff
@@ -60,6 +62,8 @@
   \setlength{\listitemcolumnwidth}{\maincolumnwidth-\listitemsymbolwidth}%
   \setlength{\doubleitemcolumnwidth}{\maincolumnwidth-\hintscolumnwidth-\separatorcolumnwidth-\separatorcolumnwidth}%
   \setlength{\doubleitemcolumnwidth}{0.5\doubleitemcolumnwidth}%
+  \setlength{\tripleitemcolumnwidth}{\maincolumnwidth-\hintscolumnwidth-\hintscolumnwidth-\separatorcolumnwidth-\separatorcolumnwidth-\separatorcolumnwidth}%
```

Comment:
Hi @jalopezg-git ,
thanks - as always 😀 - for your review. I am happy to get some tasks done here. 
>> (I saw the answer below when reading the code - perhaps it's worth adding a comment here).

I tried to follow the logic of `\cvdoubleitem`. There is also a re-definition of `\doubleitemcolumnwidth` in the following line. 
Now as you pointed it out I also see that this could be confusing, because one would think that the first definition should be the final result of `\tripleitemcolumnwidth`. 
Unfortunately I am a bit stuck here in this view at the moment. But maybe there you could come into play with a fresh new sight:
Which kind of comment would have helped you here? I am glad to insert one, but right at the moment I am a bit lost what to write for clarification. 😅

>Other than that, cannot we just use `3\separatorcolumnwidth` instead of `-\separatorcolumnwidth-\separatorcolumnwidth-\separatorcolumnwidth`?

I agree, maybe I should have thought about using a factor earlier... 
Looks more like a simplified equation now. 🤣

Reply:
> I tried to follow the logic of `\cvdoubleitem`. There is also a re-definition of `\doubleitemcolumnwidth` in the following line. Now as you pointed it out I also see that this could be confusing, because one would think that the first definition should be the final result of `\tripleitemcolumnwidth`. Unfortunately I am a bit stuck here in this view at the moment. But maybe there you could come into play with a fresh new sight: Which kind of comment would have helped you here? I am glad to insert one, but right at the moment I am a bit lost what to write for clarification. 😅

Actually, after replacing `-\separatorcolumnwidth-\separatorcolumnwidth-\separatorcolumnwidth` by `3\separatorcolumnwidth` and such, I think the intent is much clearer now and it doesn't need a further comment.

I have approved the PR as-is now; feel free to merge it!  Many thanks for working on this!

Reply:
Alright, then let us leave it as it is now. Thats also fine for me. 
Thank you for your code review and feedback! 👍